app/nagbot.py

```diff
@@ -116,11 +116,15 @@ def execute_internal(channel, dryrun):
             if len(resources_to_terminate) > 0:
                 message = f'I terminated the following {ec2_type}s: '
                 for r in resources_to_terminate:
-                    contact = sqslack.lookup_user_by_email(r.contact)
-                    message = message + r.make_resource_summary() + \
-                        f', "Terminate after"={r.terminate_after}, "Monthly Price"=' \
-                        f'{util.money_to_string(r.monthly_price)}, Contact={contact}\n'
-                    r.terminate_resource(dryrun=dryrun)
+                    response = r.terminate_resource(dryrun=dryrun)
+                    if response is not True:
+                        message = message + f"Error when attempting to terminate {r.make_resource_summary()}:" \
+                                            f" {response}\n"
```

Comment:
`f" {str(response)}\n"` can be more descriptive.

Reply:
The variable name "response" could be more descriptive?

Reply:
`f" {response}\n"` is prefixed with `Error when attempting to terminate {r.make_resource_summary()}:`, I think that should be enough information to diagnose the error (unless you're referencing the fact that response would be of type `Exception` so it needs to be typecasted to string; if that's the case, I changed the return type of `terminate_resource()` from `Exception` to `String` when a termination error occurs, so we won't need to include `str()` around `response` any more as in `str(response) since response is now a string in that case.